extensions/sts.md

```diff
@@ -283,14 +278,13 @@ section.
 
 ### Policy expiry
 
-It is possible that a client successfully receives the STS policy from a server
+It is possible that a client successfully receives an STS policy from a server
 but later on attackers begin to block all secure connection attempts from the
-client to the server until the policy expires. At that time, the client will
+client to the server until the policy expires. At that time, the client might
```

Comment:
Because it's not guaranteed. The way I've implemented it, this would only happen if the user manually edited the connection to be non-secure. If left alone it would remain on the failing secure connection. See the TODO about making this more explicit and having the redirect be stateless.